### PR TITLE
Set the --clone_model_in_keras_dist_strat to None.

### PR DESCRIPTION
**Pull request by isaprykin**

### CODE REVIEW

**Comment by isaprykin** [comment]
Uploaded the fix for PiecewiseConstantDecayWithWarmup.  tweaked benchmakrs shouldn't fail now.

File: official/resnet/keras/keras_common.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def __init__(self, batch_size, epoch_size, warmup_epochs, boundaries,
     self.compute_lr_on_cpu = compute_lr_on_cpu
     self.name = name
 
-    self.cached_learning_rate_op = None
+    self.cached_learning_rate_op_cache = {}
```
Comment (haoyuz) [inline official/resnet/keras/keras_common.py]: Either `cached_learning_rate_ops` or `learning_rate_ops_cache`?

**Comment by tfboyd** [comment]
We should still add tests to track for the next couple of weeks.

05-15 build 2.0.0-dev20190515 before  / after

**Will update with more data when test is finished**

- 9854 / 9790.87    Resnet50KerasBenchmarkSynth.benchmark_xla_8_gpu_fp16_tweaked
-  9602  Resnet50KerasBenchmarkSynth.benchmark_xla_8_gpu_fp16
-   5766  Resnet50KerasBenchmarkSynth.benchmark_8_gpu_fp16
- 2686.79 Resnet50KerasBenchmarkSynth.benchmark_8_gpu
- 2718.44 Resnet50KerasBenchmarkSynth.benchmark_8_gpu_tweaked

File: official/resnet/keras/keras_common.py (hunk 1 above)
Comment (isaprykin) [reply official/resnet/keras/keras_common.py]: Done.

**Comment by isaprykin** [comment]
Addressed the comments.

**Comment by isaprykin** [comment]
What are these lint errors for the files I didn't touch...

**Comment by tfboyd** [comment]
NP on the lint I will track them down or fix it next time we touch the file.

**Comment by isaprykin** [comment]
Woa, thanks Toby!

**Pull request merged**